.github/workflows/test_and_deploy.yml

```
@@ -32,21 +34,21 @@ jobs:
       - name: Install Rust toolchain
         uses: actions-rs/toolchain@v1
         with:
-            toolchain: stable
-            override: true
+          toolchain: stable
+          override: true
       - name: Install dependencies
-        run: pip install "maturin==1.4" "oldest-supported-numpy" pytest
+        run: pip install "maturin==$MATURIN_VERSION" "oldest-supported-numpy" pytest
```

`oldest-supported-numpy` is deprecated now, so we can pin to a late 1.x version instead.

```suggestion
        run: pip install "maturin==$MATURIN_VERSION" "numpy==1.26" pytest
```